demo-playground/start-demo-cluster.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+. $(dirname $0)/common-config.sh
+
+set -xe
+
+cabal new-build demo-playground
+
+atexit() {
+        pkill demo-playground
+}
+trap atexit EXIT
+
+start-node 0 --real-pbft
```

Comment:
Could you make the `--real-pbft` to be configurable, it will be really useful to use the others as well.

Reply:
Done in https://github.com/input-output-hk/ouroboros-network/pull/644.